### ISSUE TITLE
down-arrow does not go to paper-submenu, violation of aria best practices (a11y issue)

### ISSUE THREAD

**Issue by godinger** [PolymerElements/paper-menu#117]
### Description

Down arrow cannot be used to navigate to a paper-submenu from elements (paper-item or other paper-submenus) in its parent paper-menu. Instead, user must use the tab key. This does not comply with wai-aria-practices for a menubar (https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html#).

### Expected outcome

Down arrow navigates to paper-submenu.

### Actual outcome

Down arrow

### Live Demo
See Sub-menu in the demo for paper-menu at:
https://www.webcomponents.org/element/PolymerElements/paper-menu/demo/demo/index.html

### Steps to reproduce
Click on the item "paper-menu"
Press down arrow.

### Browsers Affected
Only tested on Chrome.
